### ISSUE TITLE
AWS Lambda with Python 3.9 (3.10, 3.11) not supported

### ISSUE THREAD

**Issue by antonpirker** [getsentry/sentry-python#1407]
### How do you use Sentry?

Sentry Saas (sentry.io)

### Version

AWS Lambda

### Steps to Reproduce

1. I create a AWS Lambda function based on Python 3.9 in the AWS Console.
2. I create a new project in sentry.io and in the `Serverless` tab I select `AWS LAMBDA (PYTHON)`.
3. I add the Sentry Cloudformation Stack to my AWS account
4. I get the screen "We found 3 functions with Node or Python runtimes". 
5. In the list of the Lambda functions my function based on Python 3.8 is visible, but my function based on Python 3.9 is missing. 

### Expected Result

Being able to select my Python 3.9 Lambda function to add the Sentry integration.

### Actual Result

The Python 3.9 function is not in the list of AWS Lambda functions I can enable Sentry for.

**Comment by chapayevdauren** [comment]
Same for me, AWS Lambda with Python3.9 not working but other functions with Python3.8 working fine

**Comment by ragelo** [comment]
Same for me, AWS Lambda, Python3.9, layer `arn:aws:lambda:us-east-1:943013980633:layer:SentryPythonServerlessSDK:6`

**Comment by van4oza** [comment]
UP! installed without layer, just sentry_sdk.init. Not capturing exceptions!!

**Comment by Angelodaniel** [comment]
I would love to have supportability for more than just 3.9 as Python is already on 3.11.4

**Comment by antonpirker** [comment]
Yes, this is kind of embarrassing by now. I will try to make time for this.

**Comment by oakhan3** [comment]
Hi, is there any update on this? Currently trying to use Sentry with python 3.10 on AWS Lambda and running into the same issue. Thank you

**Comment by antonpirker** [comment]
No update right now sorry. But I will bring this up in our internal daily today, so we can find time to make this happen.

**Comment by trovafer** [comment]
I can see in the logs that the integration gets disabled.

```
[sentry] WARNING: Not running in AWS Lambda environment, AwsLambdaIntegration disabled (could not find handle_event_request)
```

I tried using LoggingIntegration. Unfortunately, it doesn't work either. It appears the event is sent, but it does not show up in Sentry.

```

[sentry] DEBUG: Sending event, type:null level:error event_id:26ab7befaa66435aa37e915105250098 project:4505952607141888 host:o1168346.ingest.sentry.io
```

Any suggestion until this is implemented?

**Comment by antonpirker** [comment]
Hey @trovafer !
Do you get the `WARNING: Not running in AWS Lambda environment` message when running an actual Lambda function in AWS? (what python runtime are you using?)

**Comment by trovafer** [comment]
@antonpirker I did. But I found out a typo in the package version in our requirements.txt.
We were using version `1.3.1` rather than `1.31.0`.

I updated to the latest version `1.32.0` and it appears to be working now.

I'm using `python3.9`

**Comment by antonpirker** [comment]
Ok, thanks for letting us know.

**Comment by sentrivana** [comment]
Closing, this was released with 1.33.0.
